### DISCUSSION TITLE
Referencing cross-platform .svg ressources in AXML and Android Custom Views

### DISCUSSION THREAD

**Discussion by jurganson** [dotnet/maui#19750] [Q&A]
During migration from Xamarin to MAUI, I am in the process of switching from .png icons to .svg icons, I have run into an issue where my .axml pages (used for some custom Android handlers) used to reference in the views: android:src="@drawable/someResource" or even in code-behind where it is referenced through ResourceDesigner

However now, going to .svg, I would like to remove these resources and use the .svg instead. Is this possible, to somehow let MAUI generate the android drawable resources from "cross-platform" svg's?

**Comment by jurganson** [comment]
Bump 👋

No one has had any similar issues?

**Reply by danielmeza** [reply]
You are not alone, I'm with you in this suffering, should we take this to the Men's Council?
